GDJS/Runtime/runtimegame.js

```diff
@@ -212,6 +216,14 @@ gdjs.RuntimeGame.prototype.getInitialObjectsData = function() {
   return this._data.objects || [];
 };
 
+/**
+ * Get the Debugger Client.
+ * @return {?gdjs.IDebuggerClient} The Debugger (or null if not present).
+ */
+gdjs.RuntimeGame.prototype.getDebugger = function() {
```

Comment:
I would avoid adding this if it's not used in the PR?

Reply:
ok

Reply:
It's good practice in general to avoid adding something if it's not used. If you have a use case of needing this outside the PR, I'm not totally against, but it's a new public interface (i.e: will need to be available forever to avoid breaking compatibility in the future), so I'm cautious about adding these.